packages/where/index.js

```diff
@@ -24,28 +24,25 @@ const whereEndoStateWindows = env => {
   if (env.HOMEDRIVE !== undefined && env.HOMEPATH !== undefined) {
     return `${env.HOMEDRIVE}${env.HOMEPATH}\\AppData\\Endo`;
   }
-  return '.';
+  return 'Endo';
 };
 
 /**
  * @type {typeof import('./types.js').whereEndoState}
  */
 export const whereEndoState = (platform, env) => {
```

Comment:
The comment atop this file is sort of disconnected. Is it better to add something to the exported functions?

If not, use [@file](https://jsdoc.app/tags-file.html)?

Reply:
Added comments to individual functions and removed the vestigial file doc.